idaes/models/properties/modular_properties/eos/ceos.py

```diff
@@ -227,12 +228,24 @@ def func_b(m, j):
         )
 
         if mixing_rule_a == MixingRuleA.default:
+            units = b.params.get_metadata().derived_units
```

Comment:
As a heads up, you might want to talk to @viiibhav to coordinate your PR with his (https://github.com/IDAES/idaes-pse/pull/977) - there are pretty much guaranteed to conflict, so it would help to at least be aware of what each of you are doing.